README.md

```diff
@@ -4,7 +4,7 @@ Simple cross-platform plug-in that allows you to pick files and work with them.
 
 The original project can be found [here](https://github.com/Studyxnet/FilePicker-Plugin-for-Xamarin-and-Windows/), but seems abandoned, this one was forked and further developed.
 
-### The future: [Xamarin.Essentials](https://docs.microsoft.com/de-de/xamarin/essentials/)
+### The future: [Xamarin.Essentials](https://docs.microsoft.com/en-us/xamarin/essentials/)
```

Comment:
```suggestion
### The future: [Xamarin.Essentials](https://docs.microsoft.com/xamarin/essentials/)
```

Reply:
If you leave it out entirely, it will automatically select the right one for the user :)